content/articles/index.md

```diff
@@ -0,0 +1,266 @@
+![hero](/core-concepts-of-redux-store-management/hero.png)
+
+#### Intoduction
+
+Welcome to the Core Concept of Redux Store Management article!. In this article, I will be discussing the concepts and essentials of Redux store management. By the time you finished reading this article, you should be confident to tell what Redux is all about and how it is been used.
+
+#### Keys Takeaway 
+In this article, I will be covering these sub-topics been the most essentials of the Redux itself. Join me to explore more on this journey.
+
+* What is Redux and Store Management?
+* How to set up the Redux Store.
+* Concepts of Action and Reducers in Redux Data Flow.
+* Connecting an application with Redux Store.
+* Finally, How to make asynchronous calls with Redux Thunk.
+
+### What is Redux and Store Management?
+Generally speaking, Redux is said to be a JavaScript library that depicts how the state of the application should be managed and accessed. Redux manages every application state, by providing a centralized store that needs to be accessed throughout the application. There are many libraries like flux, ember, etc but Redux data flow is clear and easy to integrate with. What is the application state?
```

Comment:
This paragraph seems a little bit long. Try to break the sentences to make them shorter.
Let each paragraph explain one concept for clear understanding.